### DISCUSSION TITLE
Lists of batches instead of just batches

### DISCUSSION THREAD

**Discussion by BlenderNeko** [comfyanonymous/ComfyUI#480] [Ideas]
It might be beneficial to have latents be lists of batches (i.e. LxNxCxHxW) where each batch in the list is processed sequentially. along with a rebatch node that changes the batch size (and probably some combine tiles node or something), would give users explicit control over tiling and build on top of it without e.g. needing to create a tiled sampler. Thoughts?

This might break custom nodes already out there, though maybe with some smart checking we could support lists and non lists.

**Comment by comfyanonymous** [comment]
I have been thinking of doing this but instead of forcing nodes to support it I would add it to the execution.py and make it execute the same node for each element in the list so that way no node would have to be rewritten.

**Comment by omar92** [comment]
i think the project need a dev branch so if you do such change people can adapt there stuff before the change is released to public
